doc/source/cluster/usage-stats.rst

```diff
@@ -0,0 +1,52 @@
+.. _ref-usage-stats:
+
+Usage Stats Collection
+======================
+
+Starting in Ray 1.13, Ray collects usage stats data by default (guarded by an opt-out prompt).
+This data will be used by the open-source Ray engineering team to better understand how to improve our libraries and core APIs, and how to prioritize bug fixes and enhancements.
+For more context, please refer to this `RFC <https://github.com/ray-project/ray/issues/20857>`_.
+
+What data is collected?
+-----------------------
+We collect non-sensitive data that helps us understand how Ray is used (e.g., which Ray libraries are used).
+Personally identifiable data is never collected. Please check ``UsageStatsToReport`` in `usage_lib.py <https://github.com/ray-project/ray/blob/master/python/ray/_private/usage/usage_lib.py>`_ to see the data we collect.
```

Comment:
Can you document this (render it on the docs) and then link to it, instead of linking to a file here?